semgrep_output_v1.atd

```diff
@@ -209,8 +218,7 @@ type cli_match_extra = {
   (* fields coming from the rule *)
```

Comment:
maybe you can update the comment here as they are not coming directly from the rule now.